src/js/modules/recruitment/constants.js

```diff
@@ -0,0 +1,177 @@
+        source: "Amy Huang"
+      }
+    ],
+    link: null
+  },
+  {
+    name: "Opinions",
+    imageLink: null,
+    description: `The Opinions section of the Spectator focuses on analyzing and dissecting important, complex, 
+    and relevant issues happening in our Stuyvesant community and in the world beyond. 
+    In each issue, a number of writers across all grades give their opinions on subjects ranging from politics to economics and from technology to culture. 
+    The Opinions section often includes Point-Counter-Points and spreads that reflect a diverse set of beliefs and perspectives, 
+    and use their stances to uncover a deeper understanding of the subject at hand. 
+    The overarching goal is to create engaging, substantive, and relevant content to provide a closer look at the world.`,
+    quotes: [
+      {
+        text: `Joining the Spectator Opinions Department has been the best decision I have made freshman year here at Stuyvesant. 
+        Throughout the half of a year on the department, I’ve had the opportunity to share many of my opinions that I only told my close friends 
+        or family to a wider audience. With writing drafts and coming up with arguments, I’ve also seen myself grow as a writer, allowing me to support my 
+        arguments more effectively. As teenagers, it’s easy to feel like our voices do not matter as much as adults, 
+        but this department has allowed for many students’ voices to be recognized, and I find that very special.`,
+        source: "Angela Wong"
+      }
+    ],
+    link: null
+  },
+  {
+    name: "Copy",
+    imageLink: null,
+    description: `The Copy department makes sure The Spectator is free of all grammatical errors! 
+    Being a part of this department means honing your grammar skills and meeting other grammar nerds. We send out articles to edit roughly every two weeks, 
+    and only expect members to edit one or two articles per issue. 
+    We are a low-commitment department and don’t have many meetings, so balancing this department and other extracurriculars won’t be a problem.`,
+    quotes: [
+      {
+        text: `Being a part of Spec Copy is incredibly satisfying! I've always been a grammar nerd, 
+        so being able to use that to my advantage to help out The Spectator is absolutely a win-win situation. 
+        I love that it also means getting to see the various pieces before the final issue is released. 
+        We have some really talented writers, and being able to polish their work and contribute to such a great publication is a privilege.`,
+        source: "Tasfia Bashar"
+      }
+    ],
+    link: null
+  },
+  {
+    name: "Web",
+    imageLink: null,
+    description: `The Web department is responsible for designing and updating The Spectator's website, www.stuyspec.com. 
+    Web teaches how to develop and maintain a server using Ruby on Rails, a website using Javascript and React, and other tools using modern languages like Go. 
```

Comment:
server != API

Reply:
"using..." is a little weird imo. I'd say "Web teaches how to develop and maintain a full stack application using tools such as Ruby on Rails, React, JavaScript (ES6+) and Go"